files/build_templates/buffers_config.j2

```diff
@@ -21,6 +21,10 @@ def
 {%-     set switch_role      = '' %}
 {%- endif -%}
 
+{%- if DEVICE_NEIGHBOR_METADATA is not defined %}
```

Comment:
how come this change related to sn3700 support?
if this is a required fix it should be regardless of sn3700 PR

Reply:
@liat-grozovik this should be a fix, but for now we are using it as WA.

Reply:
in this case it should not go to the formal PR